ibm_catalog.json

```diff
@@ -78,78 +117,128 @@
               ]
             },
             {
-              "key": "use_existing_resource_group"
-            },
-            {
-              "key": "resource_group_name"
-            },
-            {
-              "key": "prefix",
-              "required": true,
-              "description": "The prefix to add to all resources that this solution creates. To not use any prefix value, you can enter the string `__NULL__`."
-            },
-            {
-              "key": "existing_monitoring_crn"
-            },
-            {
-              "key": "existing_kms_instance_crn",
-              "required": true
+              "key": "existing_scc_instance_crn"
```

Comment:
I would priortise putting inputs that are related to new instance creation first (since that is the default). So suggest to place this further down